src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -198,6 +200,22 @@ private async Task<string> UploadArchiveToAws(string bucketName, string archiveP
         return archiveBlobUrl;
     }
 
+    private async Task<string> UploadArchiveToGithub(string org, string archivePath)
+    {
+#pragma warning disable IDE0063
+        await using (var archiveData = _fileSystemProvider.OpenRead(archivePath))
+#pragma warning restore IDE0063
+        {
+            var isMultipart = archiveData.Length > 100 * 1024 * 1024; ; // Determines if stream size is greater than 100MB
```

Comment:
two things:
1. could you use a constant for the size limit, for example `STREAM_SIZE_LIMIT`
2. there is an extra semicolon :)